docs/docs/hydroflow/syntax/surface_data.mdx

```diff
@@ -41,4 +48,19 @@ in from outside the flow:
 
 <CodeBlock language="rust" showLineNumbers>{exampleCode}</CodeBlock>
 
-TODO: add the remaining sources.
\ No newline at end of file
+Sometimes we want to trigger activity based on timing, not data. To achieve this, we can use the [`source_interval()`](./surface_ops_gen.md#source_interval) operator, which takes a `Duration` `d` as an argument, and outputs a Tokio time Instant after every `d` units of time pass.
+
+## Destinations
+As duals to our data source operators, we also have data destination operators. The dest operators you'll likely use 
+most often are [`dest_sink()`](./surface_ops_gen.md#dest_sink) and [`dest_file()`](./surface_ops_gen.md#dest_file). They are fairly
+straightforward, so the best source for further information is the documentation you can find by following the links on the operator names above.
```

Comment:
Hmm.  I think of source/dest as "Hydroflow ingress/egress". You're proposing to think of it as "flow start/terminal".

I'd agree with you if and only if we rename `for_each` to `dest_XXX`: e.g. `dest_each` or `dest_map` or `dest_compute` or something like that -- emphasize that computation is a form of egress. Otherwise I think source/dest should be for ingress/egress ops.

The rename sort of makes sense -- `dest_each` is kind of dual to `source_iter -> map`; it's basically `map -> null`. I suppose the opposite of `source_iter` would be `dest_collect` into a scalar variable (but we don't have those yet).

I'll file an issue to discuss this, but until we change it I think we should leave this chapter as is.